packages/fast-development-site-react/README.md

```diff
@@ -1,2 +1,86 @@
 # FAST React development site
-This is the development site which can be used to create a react environment for a group of react or web components.
\ No newline at end of file
+This is the development site that can be used to create a React site for a library of React components. It will generate the necessary navigational elements and routing to provide a full site for the documentation and testing of components.
+
+## Installation
+`npm i --save @microsoft/fast-development-site-react`
+
+## Usage
+```
+import React from "react";
+import ReactDOM from "react-dom";
+import Site, { SiteCategory, SiteCategoryItem } from "@microsoft/fast-development-site-react";
+
+const root = document.createElement("div");
+root.setAttribute("id", "root");
+document.body.appendChild(root);
+
+function render(): void {
+    ReactDOM.render(
+        <Site
+            formChildOptions={formChildOptions}
+            onUpdateDirection={this.handleUpdateDirection}
+        >
+            <SiteTitle slot={"title"}>
+                <SiteTitleBrand>FAST</SiteTitleBrand> Documentation
+            </SiteTitle>
+            <SiteCategory
+                slot={"category"}
+                name={"Components"}
+                schema={ParagraphSchema},
+                component={Paragraph}
+            >
+                <SiteCategoryDocumentation slot={"canvas-detail-view-documentation"}>
+                    <div>
+                        Hello world
+                    </div>
+                </SiteCategoryDocumentation>
+                <SiteCategoryItem
+                    slot={"canvas-detail-view-example"}
+                    data={componentDataItem}
+                    designSystem={{ltr: this.state.direction}}
+                />
+                <SiteCategoryItem
+                    slot={"canvas-example-view"}
+                    data={componentDataItem}
+                    designSystem={{ltr: this.state.direction}}
+                />
+            </SiteCategory>
+        </Site>,
+        root
+    );
+}
+
+render();
+```
+
+### Site
+The `Site` component should be used as the wrapping component for the development site.
+
+**formChildOptions** - The options are provided to the implementation the FAST development site makes of the `@microsoft/fast-form-generator-react`, the data structure that is used is from the `Form` property `childOptions` and can be referenced [here](../fast-form-generator-react/README.md)
```

Comment:
The formChildOptions paragraph doesn't make much sense to me - suggest re-wording